### DISCUSSION TITLE
Unable to disable "pretty" exceptions

### DISCUSSION THREAD

**Discussion by NeilGirdhar** [tiangolo/typer-cli#138] [Questions] [answered]
"_TYPER_STANDARD_TRACEBACK=1 && ..."

does not work, and neither does exporting the variable.  How do we disable these excpetions?

I've also tried searching the web, searching in the source code, and searching in click.  Please let us know how to disable this, and then please also update the documentation.

**Comment by tiangolo** [comment, marked as answer]
It seems this would be a question for Typer, not Typer-CLI: https://github.com/tiangolo/typer/discussions/new?category=questions

Typer-CLI is now integrated into Typer, so everything goes in that repo.

Now, about your question (if I understand it correctly), here are the docs: https://typer.tiangolo.com/tutorial/exceptions/#disable-pretty-exceptions
